src/pages/guideline/ios/StyledBackgroundImage.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
```

Comment:
Can you move this file to components folder plz? It gets generated as route when in pages.

Reply:
yes will Do